### PR TITLE
Add method to collect labels

### LINKED ISSUE #36: Add method to audonnx.Model to collect input/output labels
As discussed in https://github.com/audeering/audonnx/pull/34#discussion_r903530109 it might be good to have a method to collect a list of labels corresponding to specific output nodes.

This could replace

```python
outputs = ['gender', 'confidence']
labels = sum([model.outputs[o].labels for o in outputs],  [])
```

with

```python
outputs = ['gender', 'confidence']
labels = model.labels(outputs=['gender', 'confidence'])
```

### PR DESCRIPTION
**Pull request by frankenjoe**
Closes #36 

image

image

### CODE REVIEW

File: audonnx/core/model.py [hunk 1]
```diff
@@ -308,6 +308,30 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return repr(self)
 
+    def labels(
+            self,
+            *,
+            outputs: typing.Union[str, typing.Sequence[str]] = None,
```
[the remark this answers is not in the document]
Comment (hagenw) [reply audonnx/core/model.py:314]: ... and for inputs we maybe should add `transforms()` then.

[the remark this answers is not in the document]
Comment (frankenjoe) [reply audonnx/core/model.py:314]: Do you see a use case to collect all transforms in a list?

[the remark this answers is not in the document]
Comment (frankenjoe) [reply audonnx/core/model.py:313]: I made `outputs` a positional argument now.

[the remark this answers is not in the document]
Comment (hagenw) [reply audonnx/core/model.py:313]: Not yet, would be more for symmetrical reasons. I would any way not propose to include it in this pull request.

So maybe we can simply skip it for now.

**Pull request merged**